kube-core/src/resource.rs

```diff
@@ -225,4 +233,8 @@ impl<K: Resource> ResourceExt for K {
     fn finalizers_mut(&mut self) -> &mut Vec<String> {
         self.meta_mut().finalizers.get_or_insert_with(Vec::new)
     }
+
+    fn creation(&self) -> Option<Time> {
+        self.meta().creation_timestamp.clone()
```

Comment:
it is possible that this method should reach into the inner chrono `DateTime` since the `k8s-openapi` one is just a [wrapper for it](https://docs.rs/k8s-openapi/latest/k8s_openapi/apimachinery/pkg/apis/meta/v1/struct.Time.html).